tests/goth_tests/test_mid_agreement_payments/requestor.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import asyncio
+import logging
+from typing import AsyncIterable
+from datetime import timedelta
+
+from yapapi import Golem, Task, WorkContext
+from yapapi.log import enable_default_logger, log_event_repr
+from yapapi.payload import vm
+
+
+async def worker(context: WorkContext, tasks: AsyncIterable[Task]):
+    async for task in tasks:
+        script = context.new_script()
+        future_result = script.run("/bin/sh", "-c", "sleep 300")
```

Comment:
please override the market strategy's debit note interval and payment timeout intervals with lower values so that you don't have to wait so long and so we can check the payment timeout interval as well

Reply:
Done, now the debit note interval is 25s; the payment timeout is 60s.